### PR TITLE
NEW! Auth Code and Number Setup 

### PR DESCRIPTION
**Pull request by rauchp**
tested on iOS and android 🤧

- back button sucks, think Carter has a higher quality for some reason
- had to make loading button, so if I could be promoted to figma dev that'd be great

### CODE REVIEW

File: civilapp/auth/SignInScreen.js [hunk 1]
```diff
@@ -172,68 +176,143 @@ export default class SignInScreen extends Component {
...
+          style={{ flex: 1 }}
+        >
+          <KeyboardText size={30} value={props.value} />
+        </TouchableHighlight>)
+    }
+    return this.renderKeyboardView(KeyboardKey)
+  }
+  renderKeyboardView = (KeyboardKey) => {
+    return (
+
```
Comment (EphemeralX) [inline civilapp/auth/SignInScreen.js]: n

File: civilapp/auth/common/SuperText.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React from 'react'
```
Comment (EphemeralX) [inline civilapp/auth/common/SuperText.js]: Good idea on SuperText to wrap this.  nits on spacing throughout this file though, sorry.

Comment (carteryu) [reply civilapp/auth/common/SuperText.js]: (▀̿Ĺ̯▀̿ ̿) This is the nit police, You're coming with us. (▀̿Ĺ̯▀̿ ̿)

Comment (rauchp) [reply civilapp/auth/common/SuperText.js]: officer please I swear it was my linter

**Pull request merged**